packages/react-ui-components/src/stories/Components/StateIndicator.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { KvStateIndicator } from '../../components';
+
+KvStateIndicator.displayName = 'KvStateIndicator';
+
+export default {
+	title: "Components/StateIndicator",
+	component: 'kv-state-indicator',
+	argTypes: {
+		color: { type: "color" },
```

Comment:
are we allowing a free choice of color in this status color indicator or should it be a predefined set of values? please verify this